### DISCUSSION TITLE
ImageInfo::LoadImage为什么使用Gdiplus::Bitmap::FromFile而不使用SHCreateStreamOnFileEx

### DISCUSSION THREAD

**Discussion by DarksKnight** [netease-im/NIM_Duilib_Framework#448] [General]
Gdiplus::Bitmap::FromFile会占用文件，导致程序加载图片后，无法删除原有图片，为什么不使用SHCreateStreamOnFileEx来处理？

**Comment by nmgwddj** [comment]
SH* 开头系列函数可能会动用 shell 相关能力，我不确定它对文件访问的性能是否会有影响，您可以尝试修改对比一下前后大量素材打开的性能。

**Reply by DarksKnight** [reply]
改了，粗略看一下没什么太大性能区别。

**Reply by DarksKnight** [reply]
后续使用的人多了过后如果有反馈。我再联系。
